packages/schematics/angular/service-worker/files/ngsw-config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "index": "/index.html",
+  "assetGroups": [{
+    "name": "app",
+    "installMode": "prefetch",
+    "resources": {
+      "files": [
+        "/favicon.ico",
+        "/index.html"
+      ],
+      "versionedFiles": [
+        "/*.bundle.css",
+        "/*.bundle.js",
+        "/*.chunk.js"
+      ]
+    }
+  }, {
+    "name": "assets",
+    "installMode": "lazy",
+    "updateMode": "prefetch",
+    "resources": {
+      "files": [
+        "/assets/**"
```

Comment:
As `assets` are lazy-loaded, is it the good place to put the `manifest.json`, which should be prefetched ? It's not an asset, it's a config file for the app like `ngsw-config.json`, so `src` would be a more consistent place to put it. But in that case, don't forget to copy it in `dist` on build.